### DISCUSSION TITLE
Rust wrapper

### DISCUSSION THREAD

**Discussion by tazz4843** [coqui-ai/STT#2091] [Show and tell]
I wrote a Rust wrapper for 🐸STT for an upcoming project, and thought I'd show it off here!

So far I've only tested it on Linux, so am not sure how it'll build on other systems and any tips there are appreciated.

https://github.com/tazz4843/coqui-stt
https://crates.io/crates/coqui-stt

It has 100% API coverage as far as I can tell, although I might've missed something somewhere.

**Comment by lissyx** [comment]
I guess you did not see that issue: https://github.com/RustAudio/deepspeech-rs/issues/48 ?

**Reply by tazz4843** [reply]
Whoop, no I did not
